server/server/data_loader/arangoload.py

```diff
@@ -720,6 +740,9 @@ def print_stage(msg):
     print_stage("Building and loading navigation from structure_dir")
     add_root_docs_and_edges(change_tracker, db, structure_dir)
 
+    print_stage("Loading child ranges from structure_dir")
+    load_child_range(db, structure_dir)
+
```

Comment:
Do we need this empty line? It seems to me like unnecessary change.

Reply:
Why? It adds nice separation of logical blocks.

Reply:
If you say so, then it's. I usually avoid such such separator.